src/app/layout/footer/footer.component.scss

```diff
@@ -0,0 +1,5 @@
+footer{
+    position:fixed;
+    bottom:0px;
+    width:100%;
+}
```

Comment:
Identation here hast more than 2 spaces.
Alos, if using the following didn't work to align it to the bottom:
```
<heade></header>
<main></main>
<footer></footer>
```
Maybe we should leave like it is, so, please, just get rid of this style as it was before.

Reply:
Done